src/main/kotlin/uk/gov/dluhc/printapi/service/PrintRequestsService.kt

```diff
@@ -18,35 +15,27 @@ class PrintRequestsService(
     private val printDetailsRepository: PrintDetailsRepository,
     private val idFactory: IdFactory,
     private val processPrintRequestQueue: MessageQueue<ProcessPrintRequestBatchMessage>,
-    private val clock: Clock,
 ) {
 
     fun processPrintRequests(batchSize: Int) {
-        batchPrintRequests(batchSize).map { (batchId, printDetails) ->
-            printDetails.map {
-                it.copy(
-                    batchId = batchId,
-                    printRequestStatuses = it.printRequestStatuses?.plus(
-                        PrintRequestStatus(Status.ASSIGNED_TO_BATCH, OffsetDateTime.now(clock))
-                    )?.toMutableList()
-                )
-            }
-        }.forEach { batch ->
-            batch.forEach { pd ->
+        batchPrintRequests(batchSize).forEach { (batchId, batchOfPrintDetails) ->
+            batchOfPrintDetails.forEach { pd ->
                 printDetailsRepository.save(pd)
                 logger.info { "Print request with id [${pd.id}] assigned to batch [${pd.batchId}]" }
             }
-            val batchId = batch.first().batchId!!
             processPrintRequestQueue.submit(ProcessPrintRequestBatchMessage(batchId))
```

Comment:
Minor : I know this is not in the scope of your PR. (Not a blocker) 
Could we move `processPrintRequestQueue.submit(ProcessPrintRequestBatchMessage(batchId))` to PrintMessagingService ? at some stage?